src/scenarios/personalization/ArrayRemoveFavorites/ArrayRemoveFavorites.tsx

```diff
@@ -0,0 +1,213 @@
+        new URL('graphql', activeProfile.personalizationServiceBaseURL).href,
+      );
+
+      const result = await apolloClient.mutate({
+        mutation: deleteArrayItemMutation,
+        variables: {
+          input: {
+            scope: 'PROFILE',
+            key: 'user_profile:favorites',
+            id: entityId,
+          },
+        },
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      if (result.errors) {
+        logger.error(
+          `method [${removeFromFavorites.name}]`,
+          'output:',
+          result.errors,
+        );
+      } else {
+        logger.log(
+          `method [${removeFromFavorites.name}]`,
+          'output:',
+          result.data,
+        );
+      }
+    } catch (error) {
+      if (error instanceof Error) {
+        logger.error(
+          `method [${removeFromFavorites.name}]`,
+          'output:',
+          error.message,
+        );
+      } else {
+        logger.error(
+          `method [${removeFromFavorites.name}]`,
+          'output:',
+          JSON.stringify(error),
+        );
+      }
+    }
+  };
+
+  return (
+    <Segment basic>
+      <Header size="huge">Array: Remove Favorite(s)</Header>
+      <Header size="small">
+        Required Services:
+        <Label>user-service</Label>
+        <Label>personalization-service</Label>
+      </Header>
+
+      <Divider />
+
+      <Container fluid>
+        <p>
+          The scenario shows an example on how to remove an item from the list
+          of user favorites.
+        </p>
+        <p>
+          In a real-life scenario any kind of list of items that is linked to a
+          user&apos;s profile can be managed (eg. favorites, watch history,
+          to-do list, etc).
+        </p>
+        <p>
+          If the user is not already signed-in, you can use one of the Sign-In
+          scenarios.
+        </p>
+      </Container>
+
+      <Divider />
+
+      <Grid divided>
+        <Grid.Column width={8}>
+          <Segment basic>
+            <Form>
+              <Form.Input
+                control={VariableSearch}
+                icon="key"
+                label="User Access Token"
+                value={userAccessToken}
+                setStateValue={setUserAccessToken}
+              />
+
+              <Form.Button
+                primary
+                onClick={async () => {
+                  fetchFavoritesByAccessToken();
+                }}
+              >
+                Fetch Favorites by Access Token
+              </Form.Button>
+
+              <Divider />
+
+              <Form.Dropdown
+                fluid
+                selection
+                label="Favorite Entity ID"
+                placeholder="Select an Entity ID"
+                options={favoriteArray.map((favorite) => {
+                  const entityValue: any = favorite?.value;
```

Comment:
Any reason for using `any` here? As per `Favorite` interface, `value` is of type `string`

Reply:
Keep this as it is.